app/Kconfig

```diff
@@ -317,6 +317,14 @@ config ZMK_RGB_UNDERGLOW_ON_START
 	bool "RGB underglow starts on by default"
 	default y
 
+config ZMK_RGB_UNDERGLOW_AUTO_OFF_IDLE
+	bool "Turn off RGB underglow when keyboard goes into idle state"
+	default y
```

Comment:
Sorry I missed this before. I would say this being default on changes previous behavior. Can we remove this for now?

```suggestion
```